File: Data Structures & Algorithms/longest-common-prefix/submission-0.cpp
class Solution 
{
public:
    string longestCommonPrefix(vector<string>& strs)
    { 
        string res  ="";
        for(int i=0;i<strs[0].size();i++)
        {
            for(int j=0;j<strs.size();j++)
            {
                if(strs[j][i]!=strs[0][i])
                {
                    return res;
                }
                else continue;
            }  
            res.push_back(strs[0][i]);
        }
        return res;

    }
};

File: Data Structures & Algorithms/transpose-matrix/submission-0.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        rows,columns=len(matrix),len(matrix[0])
        res=[[0]*rows for _ in range(columns)]
        for r in range (rows):
            for c in range (columns):
                res[c][r]=matrix[r][c]
        return res

        